Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -418,6 +418,7 @@ private class TopSitesDataSource: NSObject, UICollectionViewDataSource {
     private func createTileForSite(cell: ThumbnailCell, site: Site) -> ThumbnailCell {
         cell.textLabel.text = site.title.isEmpty ? site.url : site.title
         cell.imageWrapper.backgroundColor = UIColor.clearColor()
+        cell.backgroundImage.image = nil // Resets used cell's background image so that it doesn't get recycled when a tile doesn't update its background image.
```

Comment:
Nit: Comments usually go above the line rather than next to it (to avoid wrapping).